src/daemon/bitcoind/interface.rs

```diff
@@ -897,6 +897,31 @@ impl BitcoinD {
             Ok((_, None, _)) => self.is_in_mempool(txid),
         }
     }
+
+    /// Estimates the feerate needed for a tx to make it in the
+    /// next block. Uses estimatesmartfee and, in case it returns an
+    /// error, a default value.
+    /// The value returned is in sats/WU
+    // FIXME: Can I use u64? Wtf? I'm not sure
+    pub fn estimate_feerate(&self) -> Result<u64, BitcoindError> {
+        if let Ok(json) = self.make_node_request(
+            "estimatesmartfee",
+            &params!(Json::Number(serde_json::Number::from(1))),
```

Comment:
Can you use `2`? internally bitcoind will translate 1 to 2.

Reply:
Why would we ask for two blocks? Why would bitcoind translate 2 blocks target to 1? I don't get it 😅